src/game_inventory.cpp

```diff
@@ -1655,15 +1711,27 @@ class repair_inventory_preset: public inventory_selector_preset
     private:
         const repair_item_actor *actor;
         const item *main_tool;
+        const player &p;
 };
 
+static std::string get_repair_hint( player &p, const repair_item_actor *actor,
+                                    const item *main_tool )
+{
+    auto hint = std::string();
+    hint.append( string_format( "Tool: <color_cyan>%s</color>", main_tool->display_name() ) );
+    hint.append( string_format( " %s ", LINE_XOXO_S ) );
+    hint.append( string_format( "Skill Used: <color_cyan>%s (%d)</color>",
```

Comment:
These strings need to be translated:
```suggestion
    hint.append( string_format( _( "Tool: <color_cyan>%s</color>" ), main_tool->display_name() ) );
    hint.append( string_format( " %s ", LINE_XOXO_S ) );
    hint.append( string_format( _( "Skill Used: <color_cyan>%s (%d)</color>" ),
```